README.md

```diff
@@ -89,7 +89,7 @@ Although we strive to be fully POSIX-compliant, we recommend using Ubuntu 18.04
 
 You will need to install the following packages:
 
-    sudo apt install cmake libacl1-dev libncurses5-dev pkg-config
+    sudo apt-get install cmake libacl1-dev libncurses5-dev pkg-config
```

Comment:
I think I saw a PR which removed the "-get" and it seems it's the recommended way since some time
see https://itsfoss.com/apt-vs-apt-get-difference/

Reply:
Wasn't aware of that. Thought it's a typo! ;)